### DISCUSSION TITLE
map/surf_korovamilkbar

### DISCUSSION THREAD

**Discussion by giscus** [Kalekki/SurfHeaven_Extended#8] [Announcements]
# map/surf_korovamilkbar

Click the download icon to download surf_korovamilkbar. You can also see the top 100 records for the map as well as various other information.

https://surfheaven.eu/map/surf_korovamilkbar

**Comment by Kalekki** [comment]
Amazing S4

**Reply by timonboehm** [reply]
imagine namerevealing to thank a guy

**Comment by timonboehm** [comment]
Original S5 xdd, i could not saveloc it. Straight up T7
https://www.youtube.com/watch?v=QbAlmq1hLbE
